frontend/src/componentes/RowFarmacia.js

```diff
@@ -1,23 +1,52 @@
 import React from 'react';
 
+
+
 class RowFarmacia extends React.Component {
 
     constructor(props) {
         super(props);
         this.selectFarmacia = this.selectFarmacia.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this)
+        this.actualizar = this.actualizar.bind(this)
     }
     
     selectFarmacia() {
         this.props.selector(this.props.farmacia)
     }
 
+    actualizar() {
+        this.props.actualizarList(this.props.farmacia)
+    }
+    handleSubmit(id) {
+        fetch(`"http://localhost:8888/farmacias/" ${id}`, {
```

Comment:
Está mal la URL, si usás el backtick (la comilla torcida), no van comillas dobles. Debería ser así:
```suggestion
        fetch(`http://localhost:8888/farmacias/${id}`, {
```